### PR TITLE
Java 14

### PR DESCRIPTION
**Pull request by keeganwitt**

### CODE REVIEW

File: library/gradle [hunk 1]
```diff
@@ -18,10 +18,10 @@ Tags: 6.3.0-jre11, 6.3-jre11, jre11
 Architectures: amd64, arm64v8, ppc64le, s390x
 Directory: jre11
 
-Tags: 6.3.0-jdk13, 6.3-jdk13, jdk13
+Tags: 6.3.0-jdk14, 6.3-jdk13, jdk14
```
Comment (benkard) [inline library/gradle:21]: `s/6.3-jdk13/6.3-jdk14/`?

Comment (keeganwitt) [reply library/gradle:21]: Oops, good catch.

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index 661c82b..991d0b5 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,19 +1,19 @@
 gradle:jdk11 @ amd64
 gradle:jdk11 @ arm32v7
 gradle:jdk11 @ ppc64le
-gradle:jdk13 @ amd64
-gradle:jdk13 @ arm32v7
-gradle:jdk13 @ arm64v8
-gradle:jdk13 @ ppc64le
-gradle:jdk13 @ s390x
+gradle:jdk14 @ amd64
+gradle:jdk14 @ arm32v7
+gradle:jdk14 @ arm64v8
+gradle:jdk14 @ ppc64le
+gradle:jdk14 @ s390x
 gradle:jre11 @ amd64
 gradle:jre11 @ arm64v8
 gradle:jre11 @ ppc64le
 gradle:jre11 @ s390x
-gradle:jre13 @ amd64
-gradle:jre13 @ arm64v8
-gradle:jre13 @ ppc64le
-gradle:jre13 @ s390x
+gradle:jre14 @ amd64
+gradle:jre14 @ arm64v8
+gradle:jre14 @ ppc64le
+gradle:jre14 @ s390x
 gradle:jre @ amd64
 gradle:jre @ ppc64le
 gradle:jre @ s390x
diff --git a/_bashbrew-list b/_bashbrew-list
index a13a85d..6b8040b 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -2,26 +2,26 @@ gradle:6.3
 gradle:6.3-jdk
 gradle:6.3-jdk8
 gradle:6.3-jdk11
-gradle:6.3-jdk13
+gradle:6.3-jdk14
 gradle:6.3-jre
 gradle:6.3-jre8
 gradle:6.3-jre11
-gradle:6.3-jre13
+gradle:6.3-jre14
 gradle:6.3.0
 gradle:6.3.0-jdk
 gradle:6.3.0-jdk8
 gradle:6.3.0-jdk11
-gradle:6.3.0-jdk13
+gradle:6.3.0-jdk14
 gradle:6.3.0-jre
 gradle:6.3.0-jre8
 gradle:6.3.0-jre11
-gradle:6.3.0-jre13
+gradle:6.3.0-jre14
 gradle:jdk
 gradle:jdk8
 gradle:jdk11
-gradle:jdk13
+gradle:jdk14
 gradle:jre
 gradle:jre8
 gradle:jre11
-gradle:jre13
+gradle:jre14
 gradle:latest
diff --git a/gradle_jdk13/Dockerfile b/gradle_jdk14/Dockerfile
similarity index 97%
copy from gradle_jdk13/Dockerfile
copy to gradle_jdk14/Dockerfile
index b0b76ee..1469d8d 100644
--- a/gradle_jdk13/Dockerfile
+++ b/gradle_jdk14/Dockerfile
@@ -1,4 +1,4 @@
-FROM adoptopenjdk:13-jdk-hotspot
+FROM adoptopenjdk:14-jdk-hotspot
 
 CMD ["gradle"]
 
diff --git a/gradle_jre13/Dockerfile b/gradle_jre13/Dockerfile
deleted file mode 100644
index 638a4de..0000000
diff --git a/gradle_jdk13/Dockerfile b/gradle_jre14/Dockerfile
similarity index 97%
rename from gradle_jdk13/Dockerfile
rename to gradle_jre14/Dockerfile
index b0b76ee..58b878c 100644
--- a/gradle_jdk13/Dockerfile
+++ b/gradle_jre14/Dockerfile
@@ -1,4 +1,4 @@
-FROM adoptopenjdk:13-jdk-hotspot
+FROM adoptopenjdk:14-jre-hotspot
 
 CMD ["gradle"]
 
```

</details>

**Comment by yosifkit** [comment]
Build test of #7724; e401935c; `amd64` (`gradle`):

```console

$ bashbrew build gradle:6.3.0-jdk8
Building bashbrew/cache:0bea2993536dfe397bc4c0b99892575a52bf60879aa357d988a0b773388abb28 (gradle:6.3.0-jdk8)
Tagging gradle:6.3.0-jdk8
Tagging gradle:6.3-jdk8
Tagging gradle:jdk8
Tagging gradle:6.3.0-jdk
Tagging gradle:6.3-jdk
Tagging gradle:jdk
Tagging gradle:6.3.0
Tagging gradle:6.3
Tagging gradle:latest

$ test/run.sh gradle:6.3.0-jdk8
testing gradle:6.3.0-jdk8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gradle:6.3.0-jre8
Building bashbrew/cache:473d38c95e75fdf6d34f0f0ec912198197d5d52a64aac51b5e4c84fb2c19a9f2 (gradle:6.3.0-jre8)
Tagging gradle:6.3.0-jre8
Tagging gradle:6.3-jre8
Tagging gradle:jre8
Tagging gradle:6.3.0-jre
Tagging gradle:6.3-jre
Tagging gradle:jre

$ test/run.sh gradle:6.3.0-jre8
testing gradle:6.3.0-jre8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gradle:6.3.0-jdk11
Building bashbrew/cache:3b8b94e75ea93a178171e9074277fa875b1fb9042f0b0362e263f122d29043bd (gradle:6.3.0-jdk11)
Tagging gradle:6.3.0-jdk11
Tagging gradle:6.3-jdk11
Tagging gradle:jdk11

$ test/run.sh gradle:6.3.0-jdk11
testing gradle:6.3.0-jdk11
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gradle:6.3.0-jre11
Building bashbrew/cache:f76a82d870040d18e08ebbc2dcd5402646de2f6274eec66d6bf1f868cf85e0ae (gradle:6.3.0-jre11)
Tagging gradle:6.3.0-jre11
Tagging gradle:6.3-jre11
Tagging gradle:jre11

$ test/run.sh gradle:6.3.0-jre11
testing gradle:6.3.0-jre11
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gradle:6.3.0-jdk14
Building bashbrew/cache:d99d44f85ab5a3076706d40aadee86f9ffb528405a88ab89634de2eed4c2c63a (gradle:6.3.0-jdk14)
Tagging gradle:6.3.0-jdk14
Tagging gradle:6.3-jdk14
Tagging gradle:jdk14

$ test/run.sh gradle:6.3.0-jdk14
testing gradle:6.3.0-jdk14
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gradle:6.3.0-jre14
Building bashbrew/cache:9280b47adb230fff8062f471af5afa0e3a7d3d9fa80c4618f220bdfe2cc5424d (gradle:6.3.0-jre14)
Tagging gradle:6.3.0-jre14
Tagging gradle:6.3-jre14
Tagging gradle:jre14

$ test/run.sh gradle:6.3.0-jre14
testing gradle:6.3.0-jre14
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**